readthedocs/proxito/views/hosting.py

```diff
@@ -454,6 +454,9 @@ def _v1(self, project, version, build, filename, url, request):
             # Mainly, all the fields including a Project, Version or Build will use the exact same
             # serializer than the keys ``project``, ``version`` and ``build`` from the top level.
             "addons": {
+                "configs": {
```

Comment:
Is `configs` new? If so, I'd use either `config` or `options` instead. I wouldn't normally use the word `configs` to describe multiple configuration options.

Reply:
Yes, it's new.

I'm thinking of this field as "configurations for addons in general that don't belong to a particular addon". `config` or `options` work fine to me as well 👍🏼

Reply:
I went ahead and used `options`.